docs/root/intro/arch_overview/other_features/compression.rst

```diff
@@ -0,0 +1,21 @@
+.. _arch_overview_compression:
```

Comment:
can we actually this compression page under https://www.envoyproxy.io/docs/envoy/latest/extending/extending. I feel that after all the work that (mostly) @rojkov and I made to make compression a generic construct we should have it there.

Also that way we can say that zlib is _a_ compressor provider rather than _the_ compression provider. wdyt?

Reply:
Sure. I actually wasn’t sure on how to structure the docs. Will move it as suggested.

Reply:
@junr03, sorry, a bit confused about how to name/title the section, should it be "compression providers"?

Reply:
@rojkov and I have used the term "compression libraries" most frequently, including the field name in the [compressor](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/compressor/v3/compressor.proto#extensions-filters-http-compressor-v3-compressor)/[decompressor](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/decompressor/v3/decompressor.proto#extensions-filters-http-decompressor-v3-decompressor) filters